src/main/kotlin/com/atlassian/performance/tools/report/api/judge/NodeBalanceJudge.kt

```diff
@@ -11,37 +11,46 @@ class NodeBalanceJudge(
 ) {
     fun judge(nodeCounts: Map<String, Int>): Verdict {
         if (nodeCounts.size != criteria.nodes) {
-            return Verdict(
-                listOf<JUnitReport>(
-                    FailedAssertionJUnitReport(
-                        testMethodName(cohort),
-                        "Results are not available for some nodes. "
-                            + "There should be ${criteria.nodes} results but is ${nodeCounts.size}. "
-                            + "See node's distribution : $nodeCounts"
+            return Verdict
+                .Builder(
+                    reports =
+                    listOf<JUnitReport>(
```

Comment:
Nit: This formatting of defining argument not in the same line as it's name is new to me. Are we using it anywhere else already? If not I'd suggest to just have it in the same line like:
```
reports = listOf(
```

I'm making single comment about it, however this formatting repeats in many places of this PR and it applies to those as well.